report/4.Design/Conclusion.tex

```diff
@@ -15,11 +15,21 @@ \section{Conclusion}
 These decisions will help shape the physical aspects of the solution.
 
 
-\paragraph{Object localization}
-Unfortunately due to hardware limitations, no machine learning algorithm will be fast enough to be feasible in the project.
+\paragraph{Machine Intelligence}
+It was determined that there were two obvious applications of MI in the project.
+
+\subparagraph{Object localization}
+Unfortunately due to hardware limitations, no machine learned object localization algorithm will be fast enough to be feasible in the project.
+
 Therefore the fall back solution is to use the "Thresh Moment" algorithm, since it performs exceptionally well in terms of both performance and accuracy, while being very simple due to the utilization of the OpenCV functions.
```

Comment:
```suggestion
Therefore, the fall back solution is to use the "Thresh Moment" algorithm, since it performs exceptionally well in terms of both performance and accuracy, while being very simple due to the utilization of the OpenCV functions.
```